src/NetTopologySuite/Simplify/VWSimplifier.cs

```diff
@@ -115,7 +115,7 @@ protected override CoordinateSequence TransformCoordinates(CoordinateSequence co
                 var inputPts = coords.ToCoordinateArray();
                 Coordinate[] newPts;
                 if (inputPts.Length == 0)
-                    newPts = new Coordinate[0];
+                    newPts = Array.Empty<Coordinate>();
```

Comment:
:page_facing_up: Same basic note about behavior change as in `MinimumBoundingCircle.cs`